### PR TITLE
consume and parse kafka binlog message for java demo

### PR DESCRIPTION
**Pull request by 1156721874**
### What problem does this PR solve? 
contribute java demo that parse kafka binlog message.

### What is changed and how it works?
add  [tidb-binlog/driver/example/kafkaReader/]   catalog;
use ide run Booter.java, see README  get detail information.
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

**Comment by 1156721874** [comment]
## java解析Drainer推送到kafka里边的binlog日志

#### 环境：
tidb: v3.0.0 <br/>
drainer: v3.0.0 <br/>
kafka: kafka_2.12 1.0.0 <br/>
本地windows环境安装的protobuf版本：protoc-3.9.1-win64 <br/>
[binlog.proto](https://github.com/pingcap/tidb-tools/blob/master/tidb-binlog/slave_binlog_proto/proto/binlog.proto)是使用的官方指定的文件。

#### 使用protoc生成java文件：
protoc  --java_out=src/main/java src/main/resources/proto/descriptor.proto    --proto_path=src/main/resources/proto/  <br/>
protoc  --java_out=src/main/java src/main/resources/proto/gogo.proto    --proto_path=src/main/resources/proto/  <br/>
protoc  --java_out=src/main/java src/main/resources/proto/binlog.proto  --proto_path=src/main/resources/proto/  <br/>

#### 运行:
运行Booter启动。<br/>
指定Booter.topic、Booter.serever、Booter.offset启动即可。
 
 #### 使用kafka_reader验证kafka中的binlog的正确性
 kafka_reader文件是由go语言编译而成，用于用命令的形式解析kafka中的binlog日志，用来辅助验证的。  <br/>
 
 使用方式：<br/> 
 ./kafka_reader -offset=-1 -topic=6717826900501472462_obinlog  -kafkaAddr=192.168.138.22:9092 -commitTS=-1  -clusterID=6717826900501472462  <br/>

offset：kafka消费游标 <br/>
topic：主题名称  <br/>
kafkaAddr：kafka的broker，指定一个即可  <br/>
commitTS： binlog的commitTS，一般指定为-1  <br/>
clusterID：tidb集群id  <br/>

以上参数都是必填的。

**Comment by WangXiangUSTC** [comment]
thanks @1156721874

File: tidb-binlog/driver/example/kafkaReader/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+### java解析Drainer推送到kafka里边的binlog日志
```
Comment (zier-one) [inline tidb-binlog/driver/example/kafkaReader/README.md]: could you translate this file in English?

File: tidb-binlog/driver/example/kafkaReader/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+#### 使用protoc生成java文件：
+protoc  --java_out=src/main/java src/main/resources/proto/descriptor.proto    --proto_path=src/main/resources/proto/  <br/>
+protoc  --java_out=src/main/java src/main/resources/proto/gogo.proto    --proto_path=src/main/resources/proto/  <br/>
+protoc  --java_out=src/main/java src/main/resources/proto/binlog.proto  --proto_path=src/main/resources/proto/  <br/>
+
+#### 运行:
+运行Booter启动。<br/>
+指定Booter.topic、Booter.serever、Booter.offset启动即可。
+ 
+ #### 使用kafka_reader验证kafka中的binlog的正确性
```
Comment (zier-one) [inline tidb-binlog/driver/example/kafkaReader/README.md]: Is `kafka_reader` necessary? could we remove it?

File: tidb-binlog/driver/example/kafkaReader/README.md (hunk 1 above)
Comment (1156721874) [reply tidb-binlog/driver/example/kafkaReader/README.md]: OK，i have been   translated  [README.md](https://github.com/1156721874/tidb-tools/blob/parse-binlog-java-demo/tidb-binlog/driver/example/kafkaReader/README.md) file in English and commited.

File: tidb-binlog/driver/example/kafkaReader/README.md (hunk 2 above)
Comment (1156721874) [reply tidb-binlog/driver/example/kafkaReader/README.md]: i have been  deleted  this file and  related document  then  commited it.

**Review by zier-one — COMMENTED** [review]
LGTM

**Comment by WangXiangUSTC** [comment]
@1156721874 please sign the cla

**Comment by WangXiangUSTC** [comment]
LGTM

**Review by WangXiangUSTC — APPROVED** [review]

**Pull request merged**